docs/linux/fs.md

```diff
@@ -579,23 +579,37 @@ must be specified.
 ## Btrfs
 
 *Btrfs* is a file system based on the copy-on-write (CoW) principle resulting in
-any block update to never be written in-place. Work is ongoing to add native ZBD
-support by changing the block allocation algorithm and block IO issuing code.
+any block update to never be written in-place. Support for zoned block
+devices in *btrfs* is present, but at the moment still in experimental.
```

Comment:
"at the moment still in experimental" is ungrammatical, but I understand what is intended.

I suggest that this whole sentence ("Support... experimental.") be rewritten as follows:

*btrfs* currently supports zoned block devices, but that support is experimental.

Reply:
done